java-lotto/src/main/java/LottoGame.java

```diff
@@ -1,31 +1,38 @@
-import domain.Lotto.LottoResult;
-import domain.Lotto.LottoService;
-import domain.Lotto.LottoTicket;
-import domain.Lotto.WinningLotto;
+import domain.Lotto.*;
 import ui.OutputView;
 import ui.InputView;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 
 public class LottoGame {
-    private static final int LOTTO_PRICE = 1000;
     public static void main(String[] args) {
-
         InputView inputView = new InputView();
         OutputView outputView = new OutputView();
         LottoService lottoService = new LottoService();
 
         int purchaseAmount = inputView.getPurchaseAmount();
-        List<LottoTicket> tickets = lottoService.generateLottoTickets(purchaseAmount / LOTTO_PRICE);
-        outputView.printLottoTickets(tickets);
+        int manualTicketCount = inputView.getManualTicketCount();
+
+        LottoRequest request = new LottoRequest(
+                manualTicketCount,
+                inputView.getManualLottoNumbers(manualTicketCount)
+        );
+
+        Lottos lottos = lottoService.generateLottos(request, purchaseAmount);
 
-        LottoTicket winningTicket = inputView.getWinningTicket();
-        int BonusNumber = inputView.getBonusTicket();
-        WinningLotto winningLotto = new WinningLotto(winningTicket, BonusNumber);
+        outputView.printLottoPurchaseResult(
+                request.getManualTicketCount(),
+                lottos.getAutoTicketCount(),
+                lottos.getTickets()
+        );
```

Comment:
```suggestion
        outputView.printLottoPurchaseResult(lottos);
```
출력을 책임지는 곳에 모델을 전달해 보면 어떨까요? 이러면 유지보수 측면에서도 좋습니다.

만약 로또의 다른 속성들도 출력해 달라는 요구사항이 발생하면 `printLottoPurchaseResult` 의 메소드 시그니처가 변경되고
그만큼 출력할 값도 많다면 메소드에 전달해야하는 값도 많아지겠네요 

도메인 모델을 직접 보내는 것이 애매하다면 DTO로 만들어서 보내도되구요!